mindsdb/interfaces/agents/constants.py

```diff
@@ -174,3 +174,4 @@
 USER_COLUMN = "question"
 DEFAULT_EMBEDDINGS_MODEL_PROVIDER = "openai"
 DEFAULT_EMBEDDINGS_MODEL_CLASS = OpenAIEmbeddings
+DEFAULT_TIKTOKEN_MODEL_NAME = 'gpt-4'
```

Comment:
```suggestion
import os

DEFAULT_TIKTOKEN_MODEL_NAME = os.getenv("DEFAULT_TIKTOKEN_MODEL_NAME", "gpt-4")
```
I think this would give us the flexibility to not rebuild an image to change this value.

Reply:
Add the `import os` at the top of the file tho